content/kubelb/main/tutorials/tenants/_index.en.md

```diff
@@ -0,0 +1,67 @@
++++
+title = "Tenants"
+linkTitle = "Tenants"
+date = 2023-10-27T10:07:15+02:00
+weight = 2
++++
+
+Tenants represent the consumers of the load balancer services in the management cluster. They can be individual users, teams, or applications that have their workloads, access control, and quotas isolated by using the tenant concept in management cluster. Tenants are represented by the tenant CRD and have a dedicated namespace `tenant-<tenant-name>` in the management cluster.
+
+## Kubermatic Kubernetes Platform (Enterprise Edition Only)
+
```

Comment:
Maybe here it'd be worth to link the dedicated Kubermatic Kubernetes Platform Integration page.

Reply:
Updated